types/persist/persist.go

```diff
@@ -70,7 +77,8 @@ func (p *Persist) Equals(p2 *Persist) bool {
 		p.LoginName == p2.LoginName &&
 		p.UserProfile == p2.UserProfile &&
 		p.NetworkLockKey.Equal(p2.NetworkLockKey) &&
-		p.NodeID == p2.NodeID
+		p.NodeID == p2.NodeID &&
+		reflect.DeepEqual(p.DisallowedTKAStateIDs, p2.DisallowedTKAStateIDs)
```

Comment:
Might want to make it accept permutations of the same IDs?

Reply:
The only two places that `Equals` is used outside of tests is detecting when prefs changes for committing to disk, and for writing to the IPN bus. Given this, I think its best I leave it with the simple implementation: Equality checks might get expensive otherwise, and theres no consequence to returning false if the state IDs are in a different order.